### ISSUE TITLE
Update font awesome version UIKIT 2

### ISSUE THREAD

**Issue by miladmeidanshahi** [uikit/uikit#2923]
Hi, please update new version of font awesome in uikit 2.

**Comment by haydnjames** [comment]
I don't think this will happen. UIKIT 3 is the later stages of development so will replace UIKIT 2 soon and includes a whole new way of handling icons.

**Comment by miladmeidanshahi** [comment]
that's ok but until stable version of uikit 3 they should support uikit 2 many web site are build with version 2.

**Comment by haydnjames** [comment]
Agreed, but there is a difference between supporting and updating :)
